tests/integration/test_storage_postgresql_replica/test.py

```diff
@@ -701,26 +706,38 @@ def test_abrupt_server_restart_while_heavy_replication(started_cluster):
         database=True,
     )
     cursor = conn.cursor()
-    create_postgres_table(cursor, "postgresql_replica")
+    table_name = "postgresql_replica_697"
```

Comment:
What `697` stands for?

Reply:
the line where the test starts, I needed this because in this test file all tests have the same table name, so when you look at log, you cannot tell which logs correspond to the failed test.